lib/CL/devices/common_utils.c

```diff
@@ -616,20 +630,37 @@ pocl_cpu_validate_khr_gemm (cl_bool TransA,
 }
 #endif
 
+/**
+ * CPU_DBK_TAG
```

Comment:
What are these?

Reply:
I added these as an easy way to find all the code sections that need to be edited when adding more cpu dbks.